src/app/requestService/views/requestEditRoles.ejs

```diff
@@ -15,7 +15,7 @@
 
                         <% if (locals.validationMessages.roles !== undefined) { %>
                             <span id="validation-selected-role" class="govuk-error-message">
-                                <span class="govuk-visually-hidden">Error:</span> <%=locals.validationMessages.roles %>
+                                <span class="govuk-visually-hidden">Error:</span> <%-locals.validationMessages.roles %>
```

Comment:
Is <%- valid ejs syntax?

Reply:
this allows the html to be rendered inside the screen